ARCs/arc-0007.md

```diff
@@ -28,43 +28,43 @@ This ARC uses interchangeably the terms "throw an error" and "reject a promise w
 ### Interface `PostTxnsFunction`
 
 ```ts
-export type TxnId = string;
+export type TxnID = string;
 export type SignedTxnStr = string;
 
 export type PostTxnsFunction = (
   stxns: SignedTxnStr[],
 ) => Promise<PostTxnsResult>;
 
 export interface PostTxnsResult {
-  txnIds: TxnId[];
+  txnIDs: TxnID[];
```

Comment:
The struct defined in go-algorand has this as `TxID` not `TxnID`, should they match?


https://github.com/algorand/go-algorand/blob/bd18d0497f4a89b28fdd54811f95444adde901a4/daemon/algod/api/server/v2/generated/types.go#L704